### ISSUE TITLE
snmp asterisk script: pjsip

### ISSUE THREAD

**Issue by hb9eue** [librenms/librenms-agent#375]
Hi Team

chan_sip is depreciated. I guess everyone is using pjsip nowadays.

I altered the asterisk script to return the number of pjsip endpoints. But that is not being graphed. Instead I still get the former counters with nan counts. So I guess there is a counterpart to the asterisk snmp extend script. Could you please point me to it?

-Benoit-

**Comment by SonicJoe** [comment]
I too have the same issue. It looks like there is quite a few files to change on the back end to make this work. If I find some bandwidth, I may try my hand at it, but if you are interested here is the linked changes from when iax2 was added showing all the files that they needed to modify on the server side: https://github.com/librenms/librenms/pull/11078

**Comment by hb9eue** [comment]
Yes indeed... I once fixed it by deleting all the relevant rrd and had them re-created. But when I updated librenms all was broken again so I finally gave up :-)
But I suppose if someone submitted an easy patch containing all what is needed to count pjsip channels etc. that would be fixed easily.
Same here, if I find some spare bandwith on my voice project, I might have an deeper look into it.

**Comment by si458** [comment]
im wondering if anybody has sorted/updated the asterisk script yet?
asterisk 18 with freepbx has fully removed SIP now and only uses PJSIP
